project/assets/main/locale/es.po

```diff
@@ -16931,14 +16987,16 @@ msgid "Good job! I knew you could do it."
 msgstr "¡Buen trabajo! Sabía que podrías hacerlo."
 
 #: project/src/main/puzzle/tutorial/tutorial-spins-module.gd
+#, needs-review
 msgid ""
 "Hmm, that's not quite it. Try pressing both rotate buttons.\n"
 "\n"
-"You may need to squish first, too."
+"You need to squish first, too."
 msgstr ""
-"Hmm, no está bien. Intenta apretar ambos botones de rotación.\n"
+"Hmm, eso no es del todo correcto. Intenta apretar ambos botones de "
+"rotación.\n"
 "\n"
-"Tal vez también necesites aplastar la pieza."
+"También necesitas aplastar primero."
```

Comment:
This is significantly different from the human translation. We should preserve the original translation as it's surely better than one generated by a machine.

Reply:
Changed to `Necesitas también aplastar la pieza.` to preserve the original translator's intent.